src/components/socialLogin/FacebookConfig.js

```diff
@@ -0,0 +1,2 @@
+export const AppId = `${process.env.AppId}`;
```

Comment:
Move this to a config file in the root of the application instead and let it export an object instead